pkgs/tools/X11/jumpapp/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, perl, pandoc, fetchFromGitHub, xdotool, wmctrl, xprop, nettools }:
+
+stdenv.mkDerivation rec {
+  name = "jumpapp-${version}";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "mkropat";
+    repo = "jumpapp";
+    rev = "v${version}";
+    sha256 = "11ibh51q4vcjkz9fqyw5dy9qrkqxm42hpdccas1s6h2dk9z62kfb";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+  buildInputs = [ perl pandoc ];
+  postFixup = let
+    runtimePath = lib.makeSearchPath "bin" [ xdotool wmctrl xprop nettools perl ];
```

Comment:
what about this approach?
```
  nativeBuildInputs = [ pandoc perl ];
  buildInputs = [ xdotool wmctrl xprop nettools ];
  postFixup = let
    runtimePath = lib.makeBinPath buildInputs;
  in ...
```

This way script will be launchable from `nix-shell`, for, e.g., script development purposes.

Reply:
Neat, didn't realize that `nativeBuildInputs` can be used in this way. `perl` is both run-time and build-time dependency, so I've added it to both lists